src/packages/@ncigdc/components/PortalContainer.js

```diff
@@ -62,10 +57,8 @@ const PortalContainer = ({
       id="skip"
       role="main"
       style={{
+        minWidth: 1024,
```

Comment:
Relocated this line from 54 to sure the minimum weight is *only* on the container for the elements that currently care for the website not being responsive.